src/applications/verify/containers/VerifyApp.jsx

```diff
@@ -65,7 +65,7 @@ export class VerifyApp extends React.Component {
           <div className="row">
             <div className="columns small-12">
               <div>
-                <h1>Verify your identity</h1>
+                <h1 tabIndex="-1">Verify your identity</h1>
```

Comment:
@tmitchellgcio can you add the reasoning for this update into the description? We don't really want to set tabindex in this way

Reply:
From the proposed solution from Josh in the tagged issue. "Use a tabindex="-1" to send the focus to the h1."

Reply:
Just adding a tabindex won't send focus anywhere (also a type in the PR "tabIndex"), can you update the description with a ticket and some more info?

Reply:
I changed this to focusElement and wrote a cypress test which will verify that h1 is the focused element on load.